.eslintrc.yaml

```diff
@@ -11,12 +11,21 @@ globals:
   _: true
   $: true
   amplify: true
+  asset_service: true
```

Comment:
Instead of declaring all these global variables here, you can override ESLint rules for unused objects within the test file. So just open the test file and add an overriding rule like this on top:

```javascript
/* eslint no-unused-vars: "off" */
```